padle/padle.py

```diff
@@ -0,0 +1,537 @@
+            "\N{BLACK RIGHTWARDS ARROW}\N{VARIATION SELECTOR-16}": next_page,
+        }
+        emojis = ["\N{LEFTWARDS BLACK ARROW}\N{VARIATION SELECTOR-16}",
+                  "\N{BLACK RIGHTWARDS ARROW}\N{VARIATION SELECTOR-16}"]
+        num_pages = ceil(len(embed_texts) / 5)
+        for pageNum in range(ceil(len(embed_texts) / 5)):
+            embed = discord.Embed(title="Padle #{}".format(await self.config.num_days()), type="rich")
+            for index, text in enumerate(embed_texts[:5]):
+                embed.add_field(name="**Guess {}**:".format(index + 1 + pageNum * 5), value=text, inline=False)
+            embed.set_footer(text="Page {}/{}".format((pageNum + 1), num_pages))
+            embed_texts = embed_texts[5:]
+            embed_pages.append(embed)
+        message = await ctx.send(embed=discord.Embed(title="Padle"))
+        await self.config.user(ctx.author).edit_id.set(message.id)
+        await self.config.user(ctx.author).channel_id.set(ctx.channel.id)
+        score = ":yellow_square:"
+        if(guess_monster.monster_id == monster.monster_id):
+            await ctx.send("You got the PADle in {} guesses! Use `{}padle score` to share your score.".format(
+                len(await self.config.user(ctx.author).guesses()), prefix))
+            await self.config.user(ctx.author).done.set(True)
+            async with self.config.all_scores() as all_scores:
+                all_scores.append(str(len(await self.config.user(ctx.author).guesses())))
+            score = ":green_square:"
+        else:
+            if(points < 9):
+                score = ":orange_square:"
+            if(points < 5):
+                score = ":red_square:"
+        async with self.config.user(ctx.author).score() as scores:
+            scores.append(score)
+        start_adding_reactions(message, emojis)
+        await menus.menu(ctx, embed_pages, embed_controls, message=message, page=len(embed_pages) - 1, timeout=60 * 60)
+
+    @padle.command()
+    async def giveup(self, ctx):
+        """Give up on today's PADle"""
+        prefix = (await self.bot.get_prefix(ctx.message))[0]
+        if ctx.guild is not None:
+            return await ctx.send("You can only play PADle in DMs!")
+        if(not await self.config.user(ctx.author).start()):
+            return await ctx.send("You have not started the game of PADle yet, try `{}padle start`!".format(prefix))
+        if(await self.config.user(ctx.author).done()):
+            return await ctx.send("You have already played today's PADle!")
+        confirmation = await get_user_confirmation(ctx, "Are you sure you would like to give up?", timeout=20)
+        dbcog = await self.get_dbcog()
+        if not confirmation:
+            if confirmation is None:
+                await send_cancellation_message(ctx, "Confirmation timeout.")
+            return
+        monster = await dbcog.find_monster(str(await self.config.padle_today()), ctx.author.id)
```

Comment:
If padle_today is a monster id, you can use `dbcog.get_monster(monster_id)` which is a LOT faster.